### PR TITLE
Updating the Etcher images and text for the Live Disk support doc

### PR DESCRIPTION
**Pull request by cwsystem76**
The Etcher application has been updated significantly since the original instructions were written. I'm adding new screenshots and updating the text to match the new download site, the changes to the Etcher UI, etc.

### CODE REVIEW

File: content/live-disk.md [hunk 1]
```diff
@@ -98,27 +98,37 @@ Press the <kbd><font-awesome-icon :icon="['fab', 'pop-os']"></font-awesome-icon>
...
+![Select Pop!_OS iso](/images/live-disk-new/etcher02-filepicker.png)
 
-Next, select the drive that you want to use. If there is only one, it should automatically recognize that you have a flash drive inserted and select it.
+Now click on `Select Target`:
 
-![Etcher Flash Drive Selected](/images/live-disk/etcher-flash-selected.png)
+![Click on Select Target](images/live-disk-new/etcher03-select-target.png)
 
-Now hit the `Flash` button and watch the magic happen!
+Next, select the drive that you want to use by checking the box next to it. It should only show removeable drives by default. After the box for your USB stick is checked, click on the `Select` button:
```
Comment (garnerlogan65) [inline content/live-disk.md:115]: typo: removable

Comment (cwsystem76) [reply content/live-disk.md]: I always get that word wrong, and I meant to spellcheck it before I did the PR. Fixed.

**Review by garnerlogan65 — APPROVED** [review]
approving etcher update

**Comment by garnerlogan65** [comment]
I'm working from this branch now to revise the article. Please wait to approve until I submit my PR.

**Comment by garnerlogan65** [comment]
Please see #968 for the updated text, which includes this revision.

**Review by gangwerz — APPROVED** [review]
Changes look good to me! I think #968 will address more concerns with the source material after this is merged.

**Comment by storyaddict** [comment]
letting #968  add these changes.

**Review by garnerlogan65 — APPROVED** [review]
review approved

**Comment by leviport** [comment]
Please allow QA to review PRs before merging them.

**Pull request merged**